test/main-view-test.js

```diff
@@ -0,0 +1,78 @@
+/* global describe,beforeEach,afterEach,before,it:false */
+import environment from './env/client';
+import { equal } from 'assert';
+import sinon from 'sinon';
+import Main from '../src/js/modules/main/main';
+
+describe('Main view', () => {
+  let instance = null;
```

Comment:
is there any reason, when initializing attributes, that we set them to `null` instead of `undefined`?